src/utils/validation-errors-converter.js

```diff
@@ -22,6 +22,14 @@ function convertValidationError(error) {
       return new SchemaValidatorError(`Property '${propName}' must have value '${params.allowedValue}'.`, errorObject.dataPath, errorObject.data)
     case 'enum':
       return convertEnum(errorObject.data, errorObject.dataPath, params, propName, errorObject.schemaPath)
+    case 'keys_not_equal': {
+      const message = `Provided keys do not match expected keys: ${params.originalKeys}.`
+      return new SchemaValidatorError(message, errorObject.dataPath, errorObject.data)
+    }
+    case 'not_found':// this case use to be async
```

Comment:
`use to be`? So it means it isn't async anymore? ;) Change it rather to  `non-schema error`.